provider/grpc.go

```diff
@@ -165,6 +165,41 @@ func (g *GRPCClient) signAndSubmit(ctx context.Context, tx string, skipPreFlight
 	return response.Signature, nil
 }
 
+// signAndSubmitBatch signs the given transactions and submits them.
+func (g *GRPCClient) signAndSubmitBatch(ctx context.Context, transactions interface{}, opts SubmitOpts) (*pb.PostSubmitBatchResponse, error) {
```

Comment:
does this have to be `interface{}` type? I think it's better to require `pb.TransactionMessage` type for this and update the other calls to force this

Reply:
isn't this function only for when you have multiple tranasctions, and all endpoints that return multiple transactions are `TransactionMessage`?